### PR TITLE
Adds PreMergeBanner

### PR DESCRIPTION
**Pull request by wackerow**
## Description
Adds a pre-merge banner notification for pages flagged during site audit for containing any content requiring updating as we approach The Merge. Updating these pages will come in separate PRs, but this at least makes users aware that the content isn't fully accurate until these fixes are ready.

## Related issue
- #6281

### CODE REVIEW

File: src/components/PreMergeBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  }
+`
+
+export interface IProps {
+  className?: string
+}
+
+const PreMergeBanner: React.FC<IProps> = ({ className }) => (
+  <StyledBannerNotification shouldShow className={className}>
+    The Merge is approaching which comes with many changes to Ethereum. Some
```
Comment (minimalsm) [inline src/components/PreMergeBanner.tsx:20]: Missing comma:
```suggestion
    The Merge is approaching, which comes with many changes to Ethereum. Some
```

Slight reword, if you prefer:
```suggestion
    The Merge is approaching, and with it comes many changes to Ethereum. Some
```

**Review by minimalsm — APPROVED** [review]
LGTM except tiny typo

**Comment by wackerow** [comment]
Passing build: https://www.gatsbyjs.com/dashboard/5e02f963-3fe5-4e37-a1e7-ac05f7753a86/sites/dc7c4340-a8e9-44ed-b7be-883c70d2f02d/builds/b8cdbc09-03a5-4357-9d81-b4d8c80282c6/details

**Review by corwintines — APPROVED** [review]

File: src/components/PreMergeBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  }
+`
+
+export interface IProps {
+  className?: string
+}
+
+const PreMergeBanner: React.FC<IProps> = ({ className }) => (
+  <StyledBannerNotification shouldShow className={className}>
+    The Merge is approaching, and with it comes many changes to Ethereum. Some
```
[the remark this answers is not in the document]
Comment (wackerow) [reply src/components/PreMergeBanner.tsx:20]: Yeah, we can do that... I have another PR (#6710) that expands on this... let's just get this one in and we can add intl to the other one

**Pull request merged**